Templates/main.rb

```diff
@@ -47,22 +54,31 @@ def generate_formatter(template, interface, complexity, source_folder, destinati
     generator.save "#{destination_folder}/___FILEBASENAME___Formatter.swift"
 end
 
-def generate_view(template, interface, complexity, source_folder, destination_folder)
+def generate_ui_kit_view(template, interface, complexity, source_folder, destination_folder)
+    return unless template.generate_vc
     generator = Generator.new("#{source_folder}/___FILEBASENAME___ViewController.swift.erb", template, interface, complexity)
     generator.save "#{destination_folder}/___FILEBASENAME___ViewController.swift"
 end
 
+def generate_swift_ui_view(template, interface, complexity, source_folder, destination_folder)
+    return unless template.generate_swift_ui_view
+    generator = Generator.new("#{source_folder}/___FILEBASENAME___View.swift.erb", template, interface, complexity)
+    generator.save "#{destination_folder}/___FILEBASENAME___View.swift"
+end
+
 def generate(template, interface, complexity)
     source_folder = "#{RESOURCES}/Templates"
     destination_folder = "#{PATH}/#{template.name}.xctemplate/#{interface.name}#{complexity.name}"
     generate_xib(template, interface, complexity, source_folder, destination_folder)
     generate_sb(template, interface, complexity, source_folder, destination_folder)
     generate_interfaces(template, interface, complexity, source_folder, destination_folder)
```

Comment:
I know it's a bit out of scope, but for pure swiftui module, should we skip building interface if the complexity is simple (without interactor)?
In that case interfaces file generates empty.